io-ballerina/build.gradle

```diff
@@ -228,6 +215,20 @@ publishing {
     }
 }
 
-build {
-    dependsOn(ballerinaBuild)
-}
+ballerinaTest.dependsOn ":io-native:build"
+ballerinaTest.dependsOn ":io-native:test"
+ballerinaTest.dependsOn ":io-test-utils:build"
+ballerinaTest.dependsOn unpackJballerinaTools
+ballerinaTest.dependsOn updateTomlFile
+ballerinaTest.dependsOn initializeVariables
+ballerinaTest.finalizedBy revertTomlFile
+test.dependsOn ballerinaTest
+
```

Comment:
I think we have to add the `io-native:build`, `io-native:test` and `io-test-utild:build` dependencies on `ballerinaBuild` task. Otherwise there will be intermittent failures, I think.

Reply:
I will add them

Reply:
I added. commit id is 211245ac446f8556e340359f0fe22a5041f7e33c